lib/generators/logidze/install/functions/logidze_logger.sql

```diff
@@ -9,13 +9,22 @@ CREATE OR REPLACE FUNCTION logidze_logger() RETURNS TRIGGER AS $body$
     history_limit integer;
     debounce_time integer;
     current_version integer;
-    merged jsonb;
```

Comment:
merged wasn't used?

Reply:
yep, `merged` was extracted to `logidze_compact_history`, but decloration left: https://github.com/palkan/logidze/commit/5129ea350ace214d76627ec8dee1e69af489121d